.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```

Comment:
shall we ignore the `.idea` directory? we don't need to commit this

Reply:
yeah we should